js/main.js

```diff
@@ -105,3 +133,33 @@ document.querySelector('a').addEventListener('click', function () {
 document.querySelector('.new').addEventListener('click', function () {
   viewSwap('entry-form');
 });
+
+// Add an event listener to the ul in the entries view which does the following when an entry's pencil icon is clicked:
+$ul.addEventListener('click', pencilClick);
+
+const $title = document.querySelector('#title-text');
+const $notes = document.querySelector('#notes');
+const $entryTitle = document.querySelector('.entry-title');
+
+function pencilClick(event) {
+  // if pencil is clicked and it is the I element
+  if (event.target.tagName === 'I') {
+    const dataEntryId = event.target
+      .closest('li')
+      .getAttribute('data-entry-id');
+    // Use the viewSwap function to show the form if its true
+    viewSwap('entry-form');
```

Comment:
I would move this down to the bottom of the function, so that we only swap after all the `input`s are filled in